snapcraft/internal/project_loader/_extensions/gnome_3_28.py

```diff
@@ -25,9 +25,28 @@
 
 
 class Gnome_3_28Extension(Extension):
-    """The Gnome extension.
+    """This extension eases creation of snaps that integrate with Gnome 3.28
 
-    This extension is to be used by applications that require GTK+.
+    At build time it ensures the right build dependencies are setup and for
+    the runtime it ensures the application is run in an environment catered
+    for Gnome applications.
+
+    It sets up content interface plugs for:
+
+    \b
+    - GTK3 Themes.
+    - Common Icon Themes.
+    - Common Sound Themes
+    - The Gnome runtime libraries and utilities corresponding to 3.28.
+
+    For easier desktop integration, it also sets up each application
```

Comment:
suggestion: "sets up..." -> "configures each application with the following plugs:"